Makefile

```diff
@@ -4,17 +4,15 @@ install:
 	composer install --no-interaction
 
 clean:
-	rm -rf vendor/ dist/
+	rm -rf vendor/ dist/ composer.lock .php_cs.cache .phpunit.result.cache
 
-test: install
+test: install format
 	./vendor/bin/phpunit
-	./vendor/bin/php-cs-fixer fix --dry-run -v
 	./vendor/bin/phpmd  src/,test/ text ./phpmd.xml
-	./vendor/bin/phpcpd src/ test/
-	./vendor/bin/phploc src/
 
 coverage: install
-	./vendor/bin/phpunit --coverage-clover=dist/tests.clover
+	phpdbg -qrr ./vendor/bin/phpunit --coverage-clover dist/tests.clover
+	./vendor/bin/php-coveralls -v --coverage_clover='./dist/tests.clover' --json_path='./dist/coveralls-upload.json'
```

Comment:
iirc, we need to enable this repo in coverall.io so that it could upload the stats. It is also required to provide the API key in travis env variables.

Reply:
I already enabled it: repo keys are not needed for public repos